includes/optional-modules/class-media-partners.php

```diff
@@ -0,0 +1,522 @@
+<?php
+/**
+ * Media Partners.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Media partners.
+ */
+class Media_Partners {
+
+	/**
+	 * Initialize everything.
+	 */
+	public static function init() {
+		if ( ! function_exists( 'is_plugin_active' ) ) {
+			include_once ABSPATH . 'wp-admin/includes/plugin.php';
+		}
+
+		// If the standalone plugin is active, deactivate it and activate as a module.
+		if ( is_plugin_active( 'newspack-media-partners/newspack-media-partners.php' ) ) {
+			deactivate_plugins( 'newspack-media-partners/newspack-media-partners.php' );
```

Comment:
Apparently this is being called too early and  is causing https://github.com/Automattic/newspack-manager/blob/2993ce9f04612c581a08ac7131f6036a953560a1/includes/class-notifier.php#L61 to fail because the function `wp_get_current_user` does not exit. I suppose this coud cause other conlficts with other plugins as well...

Maybe delay it or add a try catch?

Reply:
Good catch! Should be fixed in 500cf1bb68d152388bb402741fc580842e8f3561